2.0/TransmissionTest/glTF/TransmissionTest.gltf

```diff
@@ -0,0 +1,1471 @@
+        },
+        {
+            "translation": [
+                -0.5552195310592651,
+                0.16835090517997743,
+                -9.170277870396149e-9
+            ],
+            "rotation": [
+                0.7071067690849304,
+                0.0,
+                0.0,
+                0.7071068286895752
+            ],
+            "scale": [
+                1.4414401054382325,
+                0.9999997615814209,
+                0.3821371793746948
+            ],
+            "mesh": 15,
+            "name": "MetallicTransmissionLabel"
+        },
+        {
+            "translation": [
+                -0.28168049454689028,
+                0.7958827614784241,
+                2.3661987924583629e-9
+            ],
+            "rotation": [
+                0.7071067690849304,
+                0.0,
+                0.0,
+                0.7071068286895752
+            ],
+            "scale": [
+                0.9999997019767761,
+                0.9999996423721314,
+                0.3821372091770172
+            ],
+            "mesh": 16,
+            "name": "TransmissionLabel"
+        },
+        {
+            "translation": [
+                -0.05397813767194748,
+                0.7958827018737793,
+                2.366195461789289e-9
+            ],
+            "rotation": [
+                0.7071067690849304,
+                0.0,
+                0.0,
+                0.7071068286895752
+            ],
+            "scale": [
+                0.9999997019767761,
+                0.9999998211860657,
+                0.3821372091770172
+            ],
+            "mesh": 17,
+            "name": "Transmission&Opacity"
+        },
+        {
+            "translation": [
+                0.18353992700576783,
+                0.7958827018737793,
+                2.366194573610869e-9
+            ],
+            "rotation": [
+                0.7071067690849304,
+                0.0,
+                0.0,
+                0.7071068286895752
+            ],
+            "scale": [
+                0.9999997019767761,
+                0.9999998211860657,
+                0.3821372091770172
+            ],
+            "mesh": 18,
+            "name": "TransmissionTextureLabel"
+        },
+        {
+            "translation": [
+                0.4193941354751587,
+                0.8123779892921448,
+                2.6822088994293837e-9
+            ],
+            "rotation": [
+                0.7070714235305786,
+                -0.007068855222314596,
+                0.0070688542909920219,
+                0.7070714831352234
+            ],
+            "scale": [
+                1.002103567123413,
+                1.0000001192092896,
+                0.5380522012710571
+            ],
+            "mesh": 19,
+            "name": "TransmissionTexture&OpacityLabel"
+        }
+    ],
+    "samplers": [
+        {},
+        {},
+        {},
+        {},
+        {},
+        {},
+        {}
```

Comment:
Is there some way to prevent all these empty samplers?  At most a single default sampler could be shared here.

Apologies if I'm reviewing the Dimension exporter itself more so than this particular sample, we could certainly merge this without optimizing the samplers if needed.

Reply:
Yeah, @bghgary mentioned that to me as well and it's on my list of things to fix :)